jvm/src/main/java/com/microsoft/bond/protocol/BinaryStreamReader.java

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Yes, I wrote a script to insert this to every Java file. It will be back in the next iteration.

---
In reply to: [119242235](https://github.com/Microsoft/bond/pull/465#discussion_r119242235) [](ancestors = 119242235)